### ISSUE TITLE
Error on https://f.silvergames.com/ruffle/player.php

### ISSUE THREAD

**Issue by Nokiaff14** [ruffle-rs/ruffle#11205]
# Error Info
Error name: Error
Error message: panicked at 'called `Option::unwrap()` on a `None` value', core/src/bitmap/bitmap_data.rs:618:51
Error stack:
```
Error: panicked at 'called `Option::unwrap()` on a `None` value', core/src/bitmap/bitmap_data.rs:618:51
    at O.n.wbg.__wbg_new_15d3966e9981a196 (https://f.silvergames.com/ruffle/02052023/core.ruffle.7d3ff466cfbd971981de.js:1:48914)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::hd28f65a61f685e47 (wasm://wasm/02b80182:wasm-function[8418]:0x811c79)
    at std::panicking::rust_panic_with_hook::h7f7102b82d51338f (wasm://wasm/02b80182:wasm-function[5512]:0x7338a1)
    at core::panicking::panic_fmt::he4489d678d6570d5 (wasm://wasm/02b80182:wasm-function[8271]:0x80c491)
    at core::panicking::panic::hea78c4e8b318407d (wasm://wasm/02b80182:wasm-function[7575]:0x7d2072)
    at ruffle_core::bitmap::bitmap_data::BitmapData::update_dirty_texture::h69e242bde85db84e (wasm://wasm/02b80182:wasm-function[1282]:0x3d9250)
    at <ruffle_core::library::MovieLibrarySource as ruffle_render::bitmap::BitmapSource>::bitmap_handle::h218cbf3363c21bf3 (wasm://wasm/02b80182:wasm-function[3256]:0x61f8c8)
    at <ruffle_render_wgpu::backend::WgpuRenderBackend<T> as ruffle_render::backend::RenderBackend>::register_shape::haa5f47cd0e691a1c (wasm://wasm/02b80182:wasm-function[1120]:0x389061)
    at ruffle_core::display_object::movie_clip::MovieClipData::define_shape::h127073ca34759630 (wasm://wasm/02b80182:wasm-function[3621]:0x66f2d8)
    at ruffle_core::display_object::movie_clip::MovieClip::preload::h30e5eba151b7c315 (wasm://wasm/02b80182:wasm-function[622]:0x1e2dd0)
```

# Player Info
Allows script access: false

# Page Info
Page URL: https://f.silvergames.com/ruffle/player.php?id=273
SWF URL: https://f.silvergames.com/TU-95.swf

# Browser Info
User Agent: Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Mobile Safari/537.36
Platform: Linux armv81
Has touch support: true

# Ruffle Info
Version: 0.1.0
Name: nightly 2023-05-04
Channel: nightly
Built: 2023-05-04T00:18:58.562Z
Commit: 891d1dce7e1577f950d1c54cac1b84dad3c30291
Is extension: false

# Metadata
width: 670
height: 450
frameRate: 24
numFrames: 31
swfVersion: 9
backgroundColor: #000000
isActionScript3: false
uncompressedLength: 3871229

**Comment by SN902** [comment]
matepad m5 lite '8 (2019) game works on latest ruffle demo
![Screenshot_20230525_133835_org mozilla fenix](https://github.com/ruffle-rs/ruffle/assets/38821724/13d8c9f4-8e92-4684-8ab1-6e1088ab974b)
![Screenshot_20230525_133905_org mozilla fenix](https://github.com/ruffle-rs/ruffle/assets/38821724/58d4c536-871b-480a-8516-9e3a7817465f)
but on silvergames site it panics with the same error

**Comment by n0samu** [comment]
Duplicate of #10699. This problem has already been fixed by #10908. Please contact the website owner and ask them to update their copy of Ruffle.
